dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-presto/src/main/java/org/apache/dolphinscheduler/plugin/datasource/presto/param/PrestoDataSourceProcessor.java

```diff
@@ -59,7 +62,7 @@ public BaseDataSourceParamDTO createDatasourceParamDTO(String connectionJson) {
                 prestoDatasourceParamDTO = new PrestoDataSourceParamDTO();
         prestoDatasourceParamDTO.setPort(Integer.parseInt(hostPortArray[0].split(Constants.COLON)[1]));
         prestoDatasourceParamDTO.setHost(hostPortArray[0].split(Constants.COLON)[0]);
-        prestoDatasourceParamDTO.setDatabase(connectionParams.getDatabase());
+        prestoDatasourceParamDTO.setDatabase(connectionParams.getDatabase() + "/" + connectionParams.getSchema());
```

Comment:
It's better to use `org.apache.dolphinscheduler.spi.utils.Constants#SLASH` to replace `/`.